sympy/combinatorics/free_group.py

```diff
@@ -127,6 +127,7 @@ class FreeGroup(DefaultPrinting):
     """
     is_associative = True
     is_group = True
+    is_fp_group = True
```

Comment:
I'm not sure which assignment would be more useful. It seems that testing, e.g., `is_FpGroup` could take a little less time than `isinstance` but at least now I cannot think of a situation where such tests would be needed so frequently that the difference would become meaningful. In addition, there is the problem that the attribute `is_FpGroup` would probably have to be added also elsewhere. Otherwise it could not be used without first testing its existence.